### PR TITLE
Add World Eater Beater

### PR DESCRIPTION
**Pull request by themayor897**

### CODE REVIEW

**Comment by Sabre142** [comment]
Could be moved to the 'Encounters - Enemies' section, and a trailing slash added to the end of the mod's URL.

File: masterlist.yaml [hunk 1]
```diff
@@ -22646,3 +22646,11 @@ plugins:
     clean:
       - crc: 0xBD12E363
         util: 'SSEEdit v4.0.4'
+
+  - name: 'WorldEaterBeater.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/60778' ]
```
Comment (sibir-ine) [inline masterlist.yaml]: As Sabre said, please add a trailing slash for consistency with the rest of the masterlist.

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/60778/' ]
```

File: masterlist.yaml [hunk 2]
```diff
@@ -22646,3 +22646,11 @@ plugins:
...
+    after: [ 'DeadlyDragons.esp' ]
```
Comment (sibir-ine) [inline masterlist.yaml]: LOOT should sort this way by default, but there's no harm in adding a hard rule just in case.

File: masterlist.yaml [hunk 3]
```diff
@@ -22646,3 +22646,11 @@ plugins:
...
+    inc: [ 'Dragon Combat Overhaul.esp' ]
+    clean:
+      - crc: 0xAE2F7427
+        util: 'SSEEdit v4.0.4'
```
Comment (sibir-ine) [inline masterlist.yaml]: There should be a newline at the end of the masterlist.

```suggestion
        util: 'SSEEdit v4.0.4'

```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
@themayor897 Just requires a few changes. You can also move it to the `Encounters - Enemies` section as Sabre said if you wouldn't mind, but that can always be done later.

**Comment by themayor897** [comment]
Thanks so much guys. Does that look good? Having trouble telling if there's still a new line at the end of the file or not.

**Review by sibir-ine — APPROVED** [review]

**Comment by sibir-ine** [comment]
@themayor897 Thanks!

**Pull request merged**